src/modules/CourseThumbnailAPI/utils/generateThumbnailId.ts

```diff
@@ -0,0 +1,15 @@
+import md5 from 'md5'
+
+import { thumbnailVersion } from '@/modules/CourseThumbnailAPI/constants'
+import { CourseThumbnailData } from '@/services/apollo/query/getCourse'
+
+export function generateThumbnailId(course: CourseThumbnailData): string {
+  let contents = ''
```

Comment:
Why not use string literal?

Reply:
I wanted to split it to multiple lines to make the code more readable